### ISSUE TITLE
Missing documentation

### ISSUE THREAD

**Issue by detobel36** [LoaderB0T/rxjs-typed.ts#2]
Hello,

Thank you for this project.
Is there a page where you describe all parameters like: `perLetterDelay` or `eraseDelay` ?

You also mention:
> Can be used in Angular templates directly with the [async pipe](https://angular.io/api/common/AsyncPipe)

Could you provide an example ?

**Comment by LoaderB0T** [comment]
Hi thanks for the issue, I will try to add better docs asap.

For now to answer your questions:

`perLetterDelay` is the delay between the keystrokes while typing and `eraseDelay` is the delay for backspaces (either for correcting errors or when using the `.backspace()` method.

Possible values are either a static number (in milliseconds) or an object with `min` and `max` values for a delay that is randomly chosen.

Using the `async` pipe works just as with every other observable in Angular. When you have the following code in your component ts:
```typescript
export class TestComponent implements OnInit {
  public readonly typing = new Typed();

  public ngOnInit() {
    this.typing
      .type('hello')
      .wait(500)
      .type(' world')
  }
}
```
you can use the following code in the template:
```html
<span class="typing"
  [innerHtml]="typing.text$ | async"></span>
```

make sure to have the CommonModule available in the angular Module that declares the component as it contains the async pipe.

**Comment by detobel36** [comment]
Glad to see you are still active :wink: 

Thank your for your response.
I can try to help you if I have time to read code :smiley:

**Comment by LoaderB0T** [comment]
Hi, I just updated the README with a docs section and published a new version (2.1.1) with tsdocs.
Please let me know if something is still missing.
Thanks a bunch for the feedback.

**Comment by detobel36** [comment]
Thank you verry much for your work ! <3
